### PR TITLE
Add possibility to order artifical ssn numbers

### PR DESCRIPTION
**Pull request by kivipe**
Ref: #725

Adding a new optional parameter (artifical) for FI_fi ssn provider. Default functionality
was not changed, it will provide valid Finnish SSN numbers. If set to true, it will generate
artifical ssn numbers with individual number starting with 9.

### CODE REVIEW

File: faker/providers/ssn/fi_FI/__init__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 class Provider(SsnProvider):
 
-    def ssn(self, min_age=0, max_age=105):
+    def ssn(self, min_age=0, max_age=105, artifical=False):
```
Comment (fcurella) [inline faker/providers/ssn/fi_FI/__init__.py]: typo: `s/artifical/artificial/`

Comment (kivipe) [reply faker/providers/ssn/fi_FI/__init__.py]: fixed

File: faker/providers/ssn/fi_FI/__init__.py [hunk 2]
```diff
@@ -33,7 +33,10 @@ def _checksum(hetu):
         birthday = datetime.date.today() - age
         hetu_date = "%02d%02d%s" % (
             birthday.day, birthday.month, str(birthday.year)[-2:])
-        suffix = str(self.generator.random.randrange(2, 899)).zfill(3)
+        if artifical is False:
+            suffix = str(self.generator.random.randrange(2, 899)).zfill(3)
+        else:
+            suffix = str(self.generator.random.randrange(900, 999)).zfill(3)
```
[the remark this answers is not in the document]
Comment (kivipe) [reply faker/providers/ssn/fi_FI/__init__.py]: Thanks for sharing this neat idea, fixed.

**Comment by fcurella** [comment]
Thank you! ✨

**Pull request merged**